src/disco/containers/InstallButton.js

```diff
@@ -0,0 +1,63 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import { gettext as _ } from 'core/utils';
+
+import 'disco/css/InstallButton.scss';
+import {
+  DOWNLOADING,
+  INSTALLED,
+  UNKNOWN,
+  validInstallStates as validStates,
+} from 'disco/constants';
+
+export class InstallButton extends React.Component {
+  static propTypes = {
+    handleClick: PropTypes.func,
+    handleChange: PropTypes.func,
+    guid: PropTypes.string,
+    progress: PropTypes.number,
+    slug: PropTypes.string.isRequired,
+    state: PropTypes.oneOf(validStates),
+  }
+
+  static defaultProps = {
+    state: UNKNOWN,
```

Comment:
Ah ok - yeah that makes sense I was momentarily forgetting about the role of the actions and reducers in this. Go with `status` if you like - as long as it's not just 'state' I think we're good.